scripts/translation/qaxml-pi.php

```diff
@@ -0,0 +1,71 @@
+<?php /*
++----------------------------------------------------------------------+
+| Copyright (c) 1997-2025 The PHP Group                                |
++----------------------------------------------------------------------+
+| This source file is subject to version 3.01 of the PHP license,      |
+| that is bundled with this package in the file LICENSE, and is        |
+| available through the world-wide-web at the following url:           |
+| https://www.php.net/license/3_01.txt.                                |
+| If you did not receive a copy of the PHP license and are unable to   |
+| obtain it through the world-wide-web, please send a note to          |
+| license@php.net, so we can mail you a copy immediately.              |
++----------------------------------------------------------------------+
+| Authors:     André L F S Bacci <ae php.net>                          |
++----------------------------------------------------------------------+
+
+# Description
+
+Compare processing instructions usage between two XML files.          */
+
+require_once __DIR__ . '/libqa/all.php';
+
+$argv   = new ArgvParser( $argv );
+$ignore = new OutputIgnore( $argv ); // may exit.
+$argv->complete();
+
+$list   = SyncFileList::load();
+
+foreach ( $list as $file )
+{
+    $source = $file->sourceDir . '/' . $file->file;
+    $target = $file->targetDir . '/' . $file->file;
+    $output = new OutputBuffer( "# qaxml.p" , $target , $ignore );
+
+    [ $s , $_ , $_ ] = XmlFrag::loadXmlFragmentFile( $source );
+    [ $t , $_ , $_ ] = XmlFrag::loadXmlFragmentFile( $target );
+
+    $s = XmlFrag::listNodes( $s , XML_PI_NODE );
+    $t = XmlFrag::listNodes( $t , XML_PI_NODE );
+
+    $s = extractPiData( $s );
+    $t = extractPiData( $t );
+
+    if ( implode( "\n" , $s ) == implode( "\n" , $t ) )
+        continue;
+
+    $sideCount = array();
+
+    foreach( $s as $v )
+        $sideCount[$v] = [ 0 , 0 ];
+    foreach( $t as $v )
```

Comment:
I don't understand this coding style. Usually it would be something like:
```suggestion
    foreach ($s as $v)
        $sideCount[$v] = [ 0 , 0 ];
    foreach ($t as $v)
```